Print entry preview NullPointerException handling (#13776)
JabRef/jabref: CHANGELOG.md

### Added

> - We fixed an issue where "Print preview" could throw a `NullPointerException` when no printers are available. [#13708](...)

Reviewer: Please add the full link.

> - We fixed an issue where "Print preview" could throw a `NullPointerException` when no printers are available. [#13708](...)

Reviewer 2: ```suggestion
- We fixed an issue where "Print preview" would throw a `NullPointerException` if no printers were available. [#13708](https://github.com/JabRef/jabref/issues/13708)
```

Author: added full link.
